tests/Resources/Page/ODH/Monitoring/Monitoring.resource

```diff
@@ -0,0 +1,107 @@
+*** Settings ***
+Documentation       Contains ODS Monitoring Keywords
+
+Resource            ../Prometheus/Prometheus.resource
+
+
+*** Variables ***
+${SLA_AVAILABILITY}             99.95
+${SUITE_AVAILABILITY_2M}        0
+${SUITE_AVAILABILITY_15M}       0
+${SUITE_AVAILABILITY_1H}        0
+${SUITE_AVAILABILITY_3H}        0
+
+
+*** Keywords ***
+Get Average Availability For Time Range
+    [Documentation]    Returns the average availability of ODS for a time range
+    ...    querying the Prometheus instance at ${pm_url} using token ${pm_token}.
+    ...
+    ...    returns =    avg_over_time(rhods_aggregate_availability[${time_range}]) * 100
+    [Arguments]    ${pm_url}    ${pm_token}    ${time_range}=15m    ${precision}=2
+    ${response}=    Prometheus.Run Query
+    ...    pm_url=${pm_url}
+    ...    pm_token=${pm_token}
+    ...    pm_query=avg_over_time(rhods_aggregate_availability[${time_range}])*100
```

Comment:
Does the query only look at the last `time_range` minutes? Otherwise, what is `time_range` controlling?
If my understanding is correct, we only care about a monthly SLA of 99.95, so the query returning an average availability lower than that over a 2m-3h timeframe wouldn't really tell us anything about the long term performance.
Shouldn't we be looking at the burn rate and projecting whether the unavailability would break SLA over a month?

Reply:
I think this is a discussion for the Automation meeting today. I've added an item to discuss about this PR

Reply:
As we discussed in the Automation meeting, the utility of this PR is to identify test suites &  tests cases that provoke disruptions in the availability, so we can file bugs in RHODS to address them if they are product bugs. 

In theory, if our automation covers all user interactions with RHODS and the availability is not affected after running it the monthly SLA should not be broken